src/components/BasicHead.vue

```diff
@@ -171,20 +171,10 @@ export default {
   },
 
   created () {
-    let features = ['print', 'bookmark', 'collect']
-    for(let feature of features) {
-      this.featuresMenu.push(Object.assign({}, 
-        this.tool[feature], 
-        {title: feature, click: this[this.tool[feature].click]}))
-    }
-    // console.log('featuresMenu:  ', this.featuresMenu)
+    this.$store.state.$refs = this.$refs
```

Comment:
is this needed? - this is extremely dangerous - refs point to complex objects with circular references - we should never need them to be observable

Reply:
Removed! Found a better way to do this